tests/bson-deserialize-typewrapped-binary-001.phpt

```diff
@@ -0,0 +1,105 @@
+--TEST--
+BSON serialization: type wrapped Binary
```

Comment:
Would it make more sense to title these tests based on the BSON class, or is "BSON serialization:" something you're already using?